toolkit/tools/pkg/imagecustomizerlib/customizepackages.go

```diff
@@ -40,20 +47,35 @@ var (
 )
 
 func addRemoveAndUpdatePackages(buildDir string, baseConfigPath string, config *imagecustomizerapi.OS,
-	imageChroot *safechroot.Chroot, rpmsSources []string, useBaseImageRpmRepos bool, snapshotTime string,
+	imageChroot *safechroot.Chroot, toolsChroot *safechroot.Chroot,
+	rpmsSources []string, useBaseImageRpmRepos bool, snapshotTime string,
 ) error {
 	var err error
 
 	if snapshotTime == "" {
 		snapshotTime = string(config.Packages.SnapshotTime)
 	}
 
-	err = createTempTdnfConfigWithSnapshot(imageChroot, imagecustomizerapi.PackageSnapshotTime(snapshotTime))
+	chroot := imageChroot
+	if toolsChroot != nil {
+		// Setup bind mounts to tools chroot
+		toolsChrootDir := filepath.Join(buildDir, toolsRoot)
+		source := imageChroot.RootDir()
+		target := filepath.Join(toolsChrootDir, imageRoot)
+		bindMount, err := safemount.NewMount(source, target, "", unix.MS_BIND, "", true)
```

Comment:
This only mounts the root directory. Any mounts within the directory (e.g. `/boot/efi`) won't be mounted. You likely need to do a recursive mount.

Unfortunately, I don't think you can use `MS_REC` here since you also need to unmount the child directory mounts.